src/DonorProfiles/Model.php

```diff
@@ -68,5 +68,19 @@ public function loadAssets() {
 			GIVE_VERSION,
 			true
 		);
+
+		wp_enqueue_style(
+			'give-google-font-montserrat',
+			'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap',
```

Comment:
For the sake of page load performance, we probably want to consider trimming down the number of font-weights/styles that are loaded. Last I checked, adding additional weights/styles increase page load times.

Reply:
Good point - done, ref: https://github.com/impress-org/givewp/pull/5444/commits/4361016d69e5ccabcb3f088c9bab13d007f806ee